docs/developer-docs/backend/arguments.mdx

```diff
@@ -73,19 +73,34 @@ To learn more about the `Vec` object in Typescript via Azle, refer to [the Azle
 
 </TabItem>
 
-<TabItem value="python" label="Python" default>
+<TabItem value="python" label="Python">
 
 Kybra refers to the `Vec` type to represent the equivalent of an `array` in Python. This is because `Vec` aligns with the Candid type.
 
-```
+```python
 from kybra import int32, query, Vec
 numbers = [0, 1, 2, 3]
 @query
-def get_numbers() -> Vec[int32]:
+def get_numbers(Vec[int32]) -> Vec[int32]:
     return numbers
 ```
 To learn more about `Vec` in Python via Kybra, refer to [the Kybra book reference on Vec](https://demergent-labs.github.io/kybra/reference/candid/vec.html);
 
+</TabItem>
+
+<TabItem value="bash" label="bash">
+
+You can pass a vector as an argument by explicitly specifying the canister name and method name using the `dfx` tool in the following format:
+
+```bash
+    dfx canister call canister_name method_name '(vec {})'
+```
+
+```bash
+    dfx canister call canister_name get_numbers '(vec [0,1,2,3)'
```

Comment:
```suggestion
    dfx canister call canister_name get_numbers '(vec {0;1;2;3})'
```

Reply:
Thank you for the fix here! I thought that I corrected this before but not sure why the commit did not go through!